src/main/java/cloud/distkv/distkvclient/DistkvCommandClient.java

```diff
@@ -0,0 +1,64 @@
+package cloud.distkv.distkvclient;
+
+import cloud.distkv.common.response.DistkvInitException;
+import com.distkv.client.DefaultDistkvClient;
+import com.distkv.client.commandlinetool.DistkvCommandExecutor;
+import com.distkv.common.exception.DictKeyNotFoundException;
+import com.distkv.common.exception.DistkvException;
+import com.distkv.common.exception.DistkvListIndexOutOfBoundsException;
+import com.distkv.common.exception.KeyNotFoundException;
+import com.distkv.common.exception.SortedListMemberNotFoundException;
+import com.distkv.common.exception.SortedListTopNumIsNonNegativeException;
+import com.distkv.parser.DistkvParser;
+import com.distkv.parser.po.DistkvParsedResult;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * Distkv Client for command line.
+ */
+@Component(value = "distkvCommandClient")
```

Comment:
What usage of this class?

Reply:
This class is mainly used to communicate with distkv and read data.